sdk/maps/azure-maps-traffic/CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# Release History
+
+## 1.0.0-beta.1 (2021-08-19)
```

Comment:
Please correct the release date. The content also wrong. Should be "Azure Maps Traffic" client library for Java